script.js

```diff
@@ -1,4 +1,296 @@
+//Initializing Variables
+var mode = "HOME_TURF";
+var player1Array = [];
+var player2Array = [];
+var randomArray = [];
+var myOutputValue = "";
+var randomNumberOnly = 0;
+var randomNumber1 = 0;
+var randomNumber2 = 0;
+var P1FinalNumber = 0;
+var P2FinalNumber = 0;
+var P1_Record = 0;
+var P2_Record = 0;
+var condition = "";
+var LCN_Final_Number_1 = 0;
+var LCN_Final_Number_2 = 0;
+var LCN_Final = 0;
+var P1_max = 0;
+var P2_max = 0;
+var Final_P1_VND = 0;
+var Final_P2_VND = 0;
+
+//Generate Random Number from 1 to 6
+var rollDice = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+//Check Condition for which player wins + Generating Record (Normal)
+var checkCondition = function (x, y) {
+  if (x > y) {
+    condition = "Player 1 Wins!";
+    P1_Record += 1;
+  } else if (y > x) {
+    condition = "Player 2 Wins!";
+    P2_Record += 1;
+  } else {
+    P1_Record = P1_Record;
+    P2_Record = P2_Record;
+  }
+  return condition;
+};
+
+//Check Condition for which player wins + Generating Record (LCN)
+var checkConditionLCN = function (x, y) {
+  if (x > y) {
+    condition = "Player 2 Wins!";
+    P2_Record += 1;
+  } else {
+    condition = "Player 1 Wins!";
+    P1_Record += 1;
+  }
+  return condition;
+};
+
+//Check Who is Leading
+var checkLeading = function (x, y) {
+  if (x == y) return "Draw!";
+  else if (x > y) return " Player 1 is Leading";
+  else return " Player 2 is Leading";
+};
+
+// Generating the first LCN
+var checkLCN = function (x, y) {
+  if (x < y) LCN_Final = x * 10 + y;
+  else LCN_Final = y * 10 + x;
+
+  return LCN_Final;
+};
+
+// Generating an array full of random numbers
+var generateArray = function (x, y) {
+  var counter = 0;
+  while (counter < x) {
+    randomNumberOnly = rollDice();
+    y.push(randomNumberOnly);
+    counter = counter + 1;
+  }
+  return y;
+};
+
+var findMaxCombination = function (x) {
+  var counter = 0;
+  var max = 0;
+  var max2 = 0;
+  while (counter < x[counter]) {
+    if (x[counter] > max) {
+      max2 = max;
+      max = x[counter];
+    }
+    if (x[counter] < max && x[counter] > max2) {
+      max2 = x[counter];
+    }
+    counter = counter + 1;
+  }
+
+  var FinalNumber = max * 10 + max2;
+  return FinalNumber;
+};
```

Comment:
Good work using this to automate the max dice number for the user.

What would be a better name in the function called other than `x`? Something like `diceArray`?